### ISSUE TITLE
create a subnetwork of most significant gene set per cluster

### ISSUE THREAD

**Issue by veroniquevoisin** [BaderLab/EnrichmentMapApp#542]
I have this map that contains a lot of genesets even at FDR 0.001. I looked at the results and clusters and they look interesting and robust. Is there a current implementation to do a subnetwork by choosing the most significant node per cluster? Is it in "create summary network"?
Thanks.
<img width="1412" alt="Screenshot 2024-03-06 at 2 17 26 PM" src="https://github.com/BaderLab/EnrichmentMapApp/assets/7570379/5f02ad3c-d289-4eba-830a-7f22f4b8868b">

**Comment by mikekucera** [comment]
No, there is no option to create a subnetwork from just the most significant node in each cluster. And there's no easy way to select the nodes. This needs to be added as a new feature.

**Comment by veroniquevoisin** [comment]
If I  autoannotate the network using the most significant node per cluster and then create the summary network, that should do it right?

**Comment by mikekucera** [comment]
My understanding is that you want to select the most significant node in each cluster and create a subnetwork from that, right? 

The summary network collapses each cluster into one node and creates an entirely new network. Each node in the summary network represents the aggregation of the nodes in each cluster. The summary network is not a subnetwork, it shows up in a new network collection.

**Comment by veroniquevoisin** [comment]
I solved it by using the tool to select nodes per cluster below the AA list. It worked very well and I got a nice map.
![Screenshot 2024-03-07 at 10 58 52 AM](https://github.com/BaderLab/EnrichmentMapApp/assets/7570379/99022aa7-7cf0-4a36-bef1-a48085fd8263)

The only thing is that we need to select pathway from the list to "activate the tool", correct? . I knew that I could select all pathways but not sure if users would know how to do.
